### PR TITLE
Ensure PRR approver is from "prod-readiness-approvers" team only

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: pkg/kepval/keps/validations/yaml.go [hunk 1]
```diff
@@ -219,6 +253,24 @@ func ValidateStructure(parsed map[interface{}]interface{}) error {
...
+			switch values := value.(type) {
+			case []interface{}:
+				for _, value := range values {
+					v := value.(string)
+					if len(v) > 0 && v[0] == '@' {
+						// If "@" is appeneded at the beginning, remove it.
+						v = v[1:]
+					}
+
+					index := sort. SearchStrings(prrApprovers, v)
```
Comment (johnbelamaric) [inline pkg/kepval/keps/validations/yaml.go:266]: nit: s/sort. Search/sort.Search/

Comment (wojtek-t) [reply pkg/kepval/keps/validations/yaml.go]: done

**Comment by wojtek-t** [comment]
@johnbelamaric - PTAL

**Comment by wojtek-t** [comment]
@johnbelamaric - PTAL

**Pull request merged**